src/api/vdom.js

```diff
@@ -153,7 +152,7 @@ function elementOpenStart(...args) {
 }
 
 function elementOpenEnd() {
-  const node = newElementOpen(...overrideArgs);
+  const node = newElementOpen(...overrideArgs); // eslint-disable-line no-use-before-define
```

Comment:
It's declared [here](https://github.com/skatejs/skatejs/pull/736/files#diff-9793dbfbf12d7bba475ec4c3ded46adeL31) and it has a comment telling why it's necessary. They do a similar thing in their source. Airbnb shouldn't be complaining here.